From f1154884295a4bd00d6ebcaf01fa30141145903d Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Wed, 20 Sep 2023 10:04:15 +0100
Subject: [PATCH] dt-bindings: PCI: brcmstb: add optional property -
 "brcm,tperst-clk-ms"

This property can be used to delay deassertion of external fundamental
reset, which may be useful for endpoints that require an extended time for
internal setup to complete.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 Documentation/devicetree/bindings/pci/brcm,stb-pcie.yaml | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/Documentation/devicetree/bindings/pci/brcm,stb-pcie.yaml
+++ b/Documentation/devicetree/bindings/pci/brcm,stb-pcie.yaml
@@ -77,6 +77,14 @@ properties:
       minItems: 1
       maxItems: 3
 
+  brcm,tperst-clk-ms:
+    category: optional
+    type: int
+    description: u32 giving the number of milliseconds to extend
+      the time between internal release of fundamental reset and
+      the deassertion of the external PERST# pin. This has the
+      effect of increasing the Tperst_clk phase of link init.
+
 required:
   - compatible
   - reg
